test/e2e/config/ibmcloud-e2e-powervs.yaml

```diff
@@ -60,8 +60,8 @@ variables:
 intervals:
   default/wait-controllers: ["3m", "10s"]
   default/wait-cluster: ["20m", "10s"]
-  default/wait-control-plane: ["40m", "10s"]
-  default/wait-worker-nodes: ["40m", "10s"]
+  default/wait-control-plane: ["60m", "10s"]
```

Comment:
if it taking beyond 40m then less likely it will become a node, issue might be something else, can you please test it running locally or prow job with forked repo

Reply:
The machines are getting stuck in build state, that is what I observed in the logs (no other noticeable errors). Yes they may or may not complete. I will try it on local setup and see if I can reproduce the same behaviour.